.github/workflows/DINAR-readme.yml

```
@@ -28,18 +28,21 @@ jobs:
           token: ${{ secrets.DINAR_TOKEN }}
       - name: Checkout DINAR
-        uses: actions/checkout@v2
+        # Upgraded to v4 for faster Node 20 runtime
+        uses: actions/checkout@v4
         with:
           path: DINAR
           repository: itpp-labs/DINAR-fork
           ref: master
-      - uses: actions/setup-python@v1
+      # Upgraded to v4/v5 for faster Node 20 runtime
+      - uses: actions/setup-python@v4
```

actionlint (1.7.12)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)